### ISSUE TITLE
Build error on Windows

### ISSUE THREAD

**Issue by knight9999** [jystic/unix-compat#22]
When I build this package, the following error occurs. (@ Windows 10 )
```
srcSystemPosixCompatFiles.hsc:67:7: error:
    Ambiguous occurrence ‘isSymbolicLink’
    It could refer to either ‘System.Directory.isSymbolicLink’,
                             imported from ‘System.Directory’ at srcSystemPosixCompatFiles.hsc:123:1-23
                          or ‘System.PosixCompat.Files.isSymbolicLink’,
                             defined at srcSystemPosixCompatFiles.hsc:293:1
```
Please hide `isSymbolicLInk` when importing `System.Directory`

**Comment by jystic** [comment]
I have a fix for this up on a branch at the moment, we can't hide `isSymbolicLink` as that will break on directory < 1.2.6, so will have to do explicit imports instead

**Comment by jystic** [comment]
This should be fixed in 0.4.2.0, which has now been published to Hackage, mind giving it a try?

**Comment by knight9999** [comment]
I can install the unix-compat 0.4.2.0 successfully on my windows 10 x64 environment.
Thank you very much!
